docs/UserGuide.md

```diff
@@ -367,37 +365,39 @@ Examples:
 
 * Command: `country note view c/SG`
 
-    Only country notes for Singapore will be displayed.
-
-* Command: `country note view c/IN`
-
-    Only country notes for India will be displayed.
+    Only country notes for **Singapore** will be displayed.
 
 ### Adding notes for a country: `country note add`
 
-Adds a note that is associated with a specific country.
+Adds a note that will be associated with the country which corresponds to the given country code.
 
 Format: `country note add c/COUNTRY_CODE nt/NOTE_STRING [t/TAG]...`
 
 Examples:
 
 * Command: `country note add c/SG nt/has one of the lowest corporate taxes in the world t/tax`
 
-    Adds a **note** with a **tag** for Singapore.
-
+    Adds a **note** for the country of **Singapore** with the following specifications: 
+    - Content: **_has one of the lowest corporate taxes in the world_** 
+    - Tag: **_tax_** 
+    
 * Command: `country note add c/CN nt/building good relations (guanxi) is important when conducting business here t/intercultural`
 
-    Adds a **note** with a **tag** for China.
-
+    Adds a **note** for the country of **China** with the following specifications: 
+    - Content: **_building good relations (guanxi) is important when conducting business here_** 
+    - Tag: **_intercultural_** 
+    
 * Command: `country note add c/IN nt/is world's fastest growing economy`
 
-    Adds a **note** for India.
-
+    Adds a **note** for the country of **India** with the following specifications:
+    - Content: **_is world's fastest growing economy_** 
+    - Tag: **_untagged_**
+    
 ### Editing notes for a country: `country note edit`
 
-Edits a note that is associated with a specific country at the given index based on the current view of the list panel displaying country notes. Supplying tags to the command will add the tags onto the current existing tags for that note.
+Edits a note that is associated with a specific country at the given index based on the **most recent view** of the [list panel displaying country notes](#ui-when-viewing-country). Supplying tags to the command will add the tags onto the current existing tags for that note.
 
-The country note list panel can be viewed using the command `country note view`.
+It is possible to edit a country note even when the country notes list panel is not being displayed. 
```

Comment:
Should we make this bold? Just in case. Also i realised we can delete country notes even when we haven't viewed the list panel displaying country notes lol, like directly from startup

Reply:
Not relevant now, cannot delete/edit country notes when not viewing country notes panel